3기/HyeonkyeongSeo/1주차/mission1/css/style.css

```
@@ -0,0 +1,113 @@
+/* 초기화 */
+html, body, div, h1, h2, header, nav, main, footer, .post{
+    margin: 0;
+    padding: 0;
+    border: 0;
+    outline: 0;
+    font-size: 17px;
+    background: transparent;
+    box-sizing: border-box;
+}
+
+
+
+/* 모바일 */
+
+body{
+    max-width: 320px;
+    padding: 30px 16px;
+    margin: 0 auto;
+    display: flex;
+    flex-flow: column;
+    grid-row-gap: 16px;
+    grid-column-gap: 16px;
```

CSS3 에서부터 `grid-` 라는 글자는 빠져도 되게 되었어요!
```suggestion
    row-gap: 16px;
    column-gap: 16px;
```